MediaBrowser.XbmcMetadata/Parsers/NfoParserHelpers.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Xml;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.XbmcMetadata.Savers;
+using Microsoft.Extensions.Logging;
+
+namespace MediaBrowser.XbmcMetadata.Parsers
+{
+    internal class NfoParserHelpers
```

Comment:
There's no state in this class. New'ing it up every time you need to parse XML is inefficient. It should be registered as a singleton or remove the logger and make it completely static.

Reply:
The parser itself is also recreated every time a nfo file is read. So would it be best if I made that a singleton as well?